spec.md

```diff
@@ -1066,6 +1066,28 @@ MUST follow the same style rules as other binary [comparison][] operators:
 $variable = $foo ?: 'bar';
 ```
 
+### 6.4. Operator's placement
+
+A statement that includes an operator MAY be split across multiple lines, where
+each subsequent line is indented once. When doing so, the operator MUST be
+placed at the beginning of the new line; ternaries MUST occupy 3 lines, never 2.
```

Comment:
This is contradicting 5.2, which says:

> Boolean operators between conditions MUST always be at the beginning or at the end of the line, not a mix of both.

Reply:
That's right! And this line is repeated 4 times. It would be better to drop the "`or at the end of the line`" part.

What do you think?

Reply:
Hm.  Yes, please file a new PR to remove that clause.

Reply:
https://github.com/php-fig/per-coding-style/pull/103